android/authentication.md

```diff
@@ -791,6 +834,36 @@ You can now [configure Google in your mobile app](./authentication#google-login-
 
 Note that the CLI allows you to select more than one identity provider for your app. You can also run `amplify auth update` to add an identity provider to an existing auth configuration.
 
+#### Developer Authenticated Identities with Cognito Identity
+
+In some cases, the developer has the means to authenticate the end-user. This means that the developer is the identity provider.
```

Comment:
This seems confusing, and it's not really accurate. Dev Auth means that you are using your own home rolled IdP rather than one of the industry standards.

Reply:
Suggest: With developer authenticated identities, you can register and authenticate users via your own existing authentication process, while still using Amazon Cognito to access AWS resources. Using developer authenticated identities involves interaction between the end user device, your backend for authentication, and Amazon Cognito.

Reply:
updated, please check new version